statusHandler/presenter/common.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go/core"
 )
 
-const invalidKey = "[invalid key]"
+const invalidKey = "N/A"
 const invalidType = "[not a string]"
```

Comment:
can we replace all default strings to N/A or something like this?

Reply:
done